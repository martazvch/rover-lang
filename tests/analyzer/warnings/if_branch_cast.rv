fn main() {
    _ = if false do 1.5 else {4}
    _ = if true {1} else {5.6}
    _ = if true {1.5} else {5*2}
}

