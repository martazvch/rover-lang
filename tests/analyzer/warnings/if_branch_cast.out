implicit_cast, else branch, float
implicit_cast, then branch, float
implicit_cast, else branch, float

